### PR TITLE
docs(release): updated trg template to latest version 

### PR DESCRIPTION
**Pull request by matbmoser**
#### Added
- Added the new TRG 9.01
- Added the new documentation trgs: TRG 1.05 - TRG 1.08

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [ ] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [ ] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

**Comment by matbmoser** [comment]
@evegufy @stephanbcbauer @tom-rm-meyer-ISST please check if something is missing

File: .github/ISSUE_TEMPLATE/qg-checklist.md [hunk 1]
```diff
@@ -32,6 +32,10 @@ Release Management Reference Issue: <!-- Note: Add the related product RM issue
 - [ ] [TRG 1.02](https://eclipse-tractusx.github.io/docs/release/trg-1/trg-1-2) appropriate install instructions either `INSTALL.md` or in `README.md`
 - [ ] [TRG 1.03](https://eclipse-tractusx.github.io/docs/release/trg-1/trg-1-3) appropriate `CHANGELOG.md`
 - [ ] [TRG 1.04](https://eclipse-tractusx.github.io/docs/release/trg-1/trg-1-4) editable static files
+- [ ] [TRG 1.05](https://eclipse-tractusx.github.io/docs/release/trg-1/trg-1-5) architecture docs
```
Comment (tom-rm-meyer-ISST) [inline .github/ISSUE_TEMPLATE/qg-checklist.md]: new links are wrong. Those are `05`, `06`, `07` and `08`. That's likely the reason for the wrong ordering on github page, too.

**Review by tom-rm-meyer-ISST — CHANGES_REQUESTED** [review]
Thanks! Please see comment.

File: .github/ISSUE_TEMPLATE/qg-checklist.md (hunk 1 above)
Comment (matbmoser) [reply .github/ISSUE_TEMPLATE/qg-checklist.md]: ahhh

Comment (matbmoser) [reply .github/ISSUE_TEMPLATE/qg-checklist.md]: but now what shall we fix the original links in the page or here the link? what makes more sense?

Comment (evegufy) [reply .github/ISSUE_TEMPLATE/qg-checklist.md]: align within TRG 1 to without `0` so that the ordering also works properly, I'd say

Comment (tom-rm-meyer-ISST) [reply .github/ISSUE_TEMPLATE/qg-checklist.md]: Yes, but if we already need to update something, why not making it fully consistent? The extra effort is low

Comment (evegufy) [reply .github/ISSUE_TEMPLATE/qg-checklist.md]: With fully consistent you mean for all TRGs either with or without `0`?

Comment (tom-rm-meyer-ISST) [reply .github/ISSUE_TEMPLATE/qg-checklist.md]: exactly. Being honest, I would prefer the 0 as they displayed names are also with leading `0`

Comment (matbmoser) [reply .github/ISSUE_TEMPLATE/qg-checklist.md]: shall I change the links now and we discuss this later?

Comment (evegufy) [reply .github/ISSUE_TEMPLATE/qg-checklist.md]: Fine for me to discuss this later
Also fine all TRGs with leading `0` --> maybe change after release 24.08?

**Review by evegufy — COMMENTED** [review]
@matbmoser maybe you could still remove the space at `[TRG 4.07]SPACE(h...`

File: .github/ISSUE_TEMPLATE/qg-checklist.md (hunk 1 above)
Comment (tom-rm-meyer-ISST) [reply .github/ISSUE_TEMPLATE/qg-checklist.md]: Yeah, let's make it without hurry (Y)

Comment (matbmoser) [reply .github/ISSUE_TEMPLATE/qg-checklist.md]: fixed

**Comment by matbmoser** [comment]
Done @evegufy and @tom-rm-meyer-ISST !

**Review by evegufy — APPROVED** [review]

**Review by tom-rm-meyer-ISST — APPROVED** [review]
LGTM, Raised this [issue ](https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/issues/1005) to raise awareness. Saw already other inconsistencies during changelog creation.

**Pull request merged**